README.md

```diff
@@ -26,6 +26,8 @@ Hexpm requires the PostgreSQL modules [pg_trgm](http://www.postgresql.org/docs/9
 
 This is located in the "postgresql-contrib" package, however the package name can vary depending on your operating system. If the module is not installed the ecto migrations will fail.
 
+If you want to use PostgreSQL with docker, run: `docker run --name hexpm-db -p 5432:5432 -e POSTGRES_PASSWORD=postgres -e POSTGRES_USER=postgres -d postgres:10.3`
```

Comment:
I agree, it is likely hexpm will be dockerized in the future and then we can add a postgres container to a docker-compose file or kubernetes config.

Reply:
Sounds good to me, just sent this PR because like me, other people don't keep services running in the computer if it's not being used. Just as obvious as explaining how to bootstrap the project running `iex -S mix phx.server`, the readme for me was used for a quick copy/past of commands required to get the app running locally as soon as possible. Closing this PR, thanks for the feedback o/